tests/network/non_functional/ip_persistence/conftest.py

```
@@ -0,0 +1,94 @@
+    with NetworkAttachmentDefinition(
+        name="test-bridge-network",
+        namespace=namespace.name,
+        config=config,
+        client=admin_client,
+    ) as nad:
+        yield nad
+
+
+@pytest.fixture(scope="module")
+def linux_bridge_vm_for_ip_persist(
+    ipv4_supported_cluster: bool,
```

Ruff (0.15.2)

[error] Boolean-typed positional argument in function definition

(FBT001)